Gruntfile.js

```diff
@@ -70,7 +71,14 @@ module.exports = function( grunt ) {
 				command: 'php bin/verify-version-consistency.php',
 			},
 			composer_install: {
-				command: 'if [ ! -e build ]; then echo "Run grunt build first."; exit 1; fi; cd build; composer install --no-dev -o && composer remove cweagans/composer-patches --update-no-dev -o && rm -r ' + productionVendorExcludedFilePatterns.join( ' ' ),
+				command: [
+					'if [ ! -e build ]; then echo "Run grunt build first."; exit 1; fi',
+					'cd build',
+					'composer install --no-dev -o',
+					'for symlink in $(find vendor/amp -type l); do symlinkpath=$(readlink "$symlink") && rm "$symlink" && mkdir "$symlink" && rsync -a "$(dirname "$symlink")/$symlinkpath/" "$symlink/"; done',
```

Comment:
@schlessera @pierlon This is what I came up with to fix `npm run build` after symlinks were restored. There's probably a better way!

Reply:
Perfecto! This can suffice for now :ok_hand:.